core/src/main/python/deploy.py

```diff
@@ -121,7 +122,7 @@ def __deploy_online(model, model_context, aliases):
 
     try:
         __wlst_helper.connect(admin_user, admin_pwd, admin_url)
-        deployer_utils.ensure_no_uncommitted_changes_or_edit_sessions()
+        deployer_utils.ensure_no_uncommitted_changes_or_edit_sessions(model_context.is_undo_current_edit())
```

Comment:
Deploy should do check and do deployer_utils.undo_current_edit same as update.

Reply:
added